man/puppet-db.pod

```diff
@@ -0,0 +1,91 @@
+=head1 NAME
+
+puppet db - manages PuppetDB administrative tasks
```

Comment:
Happened to notice that the style here doesn't match the style below for `puppet query`, and so wondered which might be preferable.  Not sure, but I did glance at some (perhaps inconclusive) precedents:

git - the stupid content tracker
rsync - a fast, versatile, remote (and local) file-copying tool
ls - list directory contents
man - an interface to the on-line reference manuals